client/src/e2etests/oneToOne.test.js

```diff
@@ -62,7 +62,7 @@ test("Should draw one to one nodes", async () => {
     `,
     );
 
-    const summarySelector = ".graph-overlay .title";
+    const summarySelector = ".graph-overlay.collapsed .title";
```

Comment:
This change doesn't affect behavior - it still matches the same element. Please revert